tutorials/OnnxTensorflowExport.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "# Train in Tensorflow, Export to ONNX\n",
+    "In this tutorial, we will demonstrate the complete process of training a MNIST model in Tensorflow and exporting the trained model to ONNX.\n",
+    "\n",
+    "### Training\n",
+    "\n",
+    "Firstly, we can initiate the [training script](./assets/tf-train-mnist.py) by issuing the command `python tf-train-mnist.py` on your terminal. Shortly, we should obtain a trained MNIST model. The training process needs no special instrumentation. However, to successfully convert the trained model, onnx-tensorflow requires three pieces of information, all of which can be obtained after training is complete:\n",
+    "\n",
+    "  - *Graph definition*: You need to obtain information about the graph definition in the form of GraphProto. The easiest way to achieve this is to use the following snippet of code as shown in the example training script:\n",
+    "```\n",
+    "  with open(\"graph.proto\", \"wb\") as file:\n",
+    "    graph = tf.get_default_graph().as_graph_def(add_shapes=True)\n",
```

Comment:
For some reason this indentation isn't showing up when this is rendered as markdown